NitroxModel-Subnautica/Helper/VehicleModelFactory.cs

```diff
@@ -15,13 +15,13 @@ public static VehicleModel BuildFrom(NitroxTechType techType, NitroxId construct
             switch (techType.ToUnity())
             {
                 case TechType.Seamoth:
-                    return new SeamothModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, Optional.Empty, name, hsb, health);
+                    return new SeamothModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, dockingBayId, name, hsb, health);
                 case TechType.Exosuit:
-                    return new ExosuitModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, Optional.Empty, name, hsb, health);
+                    return new ExosuitModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, dockingBayId, name, hsb, health);
                 case TechType.Cyclops:
-                    return new CyclopsModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, Optional.Empty, name, hsb, health);
+                    return new CyclopsModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, dockingBayId, name, hsb, health);
                 case TechType.RocketBase:
-                    return new NeptuneRocketModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, Optional.Empty, name, hsb, health);
+                    return new NeptuneRocketModel(techType, constructedItemId, position, rotation, interactiveChildIdentifiers, dockingBayId, name, hsb, health);
```

Comment:
Could you say/investigate if this change breaks something down the line?

Reply:
Sure!
My change doesn't do anything at all, since this factory is only used by `NitroxClient.GameLogic.Vehicles.BuildVehicleModelFrom(GameObject, TechType)`

```csharp
 return VehicleModelFactory.BuildFrom(
                    techType.ToDto(),
                    constructedObjectId,
                    gameObject.transform.position.ToDto(),
                    gameObject.transform.rotation.ToDto(),
                    childIdentifiers,
                    Optional.Empty, //<--------------
                    name,
                    hsb ?? VehicleHelper.GetDefaultColours(techType), //Shouldn't be null now, but just in case
                    health
                );
```

And we're only filling the "dockingBayId" with an Optional.Empty